pontoon/insights/migrations/0003_project_insights.py

```diff
@@ -0,0 +1,199 @@
+# Generated by Django 3.2.4 on 2021-08-24 06:45
+
+import datetime
+from django.db import migrations, models
+import pontoon.insights.models
+
+
+_activity_sql = """
```

Comment:
Note that those instructions are specific to `RunPython`, and this migration is using `RunSQL`. If you still think it's beneficial, I can split the migration into two steps.

Reply:
Sounds like we're good.

Reply:
I just tried to unapply the migration on stage and hit:
```
django.db.utils.OperationalError: cannot ALTER TABLE "insights_projectinsightssnapshot" because it has pending trigger events
```

Interestingly enough, this is the error Django docs talk about when combining schema changes and `RunPython` operations. Perhaps `RunSQL` doesn't make a difference?

*On databases that do support DDL transactions (SQLite and PostgreSQL), RunPython operations do not have any transactions automatically added besides the transactions created for each migration. Thus, on PostgreSQL, for example, you should avoid combining schema changes and RunPython operations in the same migration or you may hit errors like OperationalError: cannot ALTER TABLE "mytable" because it has pending trigger events.*

Reply:
Okay, I'll separate the data migrations into their own thing.